elfpy/data/acquire_data.py

```diff
@@ -40,52 +40,64 @@ def main(
     transactions_hyperdrive_contract = contract_interface.get_hyperdrive_contract(
         transactions_abi_file_path, contracts_url, web3
     )
+
     # get pool config from hyperdrive contract
+    # TODO should this be added to postgres to put everything in one place?
```

Comment:
Yes, ideally it would go into it's own DB that can be related to the others. The config is defines how HD was initialized, so it's needed to reproduce any bugs. You _could_ just replicate the data across rows in the pool info DB but I think the "proper" way to do it is to have it live on its own with some column that relates it to the others.

Reply:
TODO in next PR